apps/public-docsite-v9/README.md

```diff
@@ -0,0 +1,25 @@
+# [Fluent UI V9 website](https://react.fluentui.dev/)
```

Comment:
# public-docsite-v9

This app is official documentation for Fluentui react-components published to https://react.fluentui.dev